### ISSUE TITLE
Categorisation game chart broken

### ISSUE THREAD

**Issue by gbp** [mysociety/alaveteli#8301]
Google has deprecated and retired the chart API
https://support.google.com/docs/thread/251700701

Should look to move to something else or use CSS transform/rotate options to create the gauge ourself.

This is how it used to look - from the [archive.org](https://web.archive.org/web/20220526055503im_/https://chart.googleapis.com/chart?chs=250x125&cht=gom&chd=t%3A88.23):

![image](https://github.com/mysociety/alaveteli/assets/5426/934d5d33-6f42-4d39-b988-4f3324e4ee2a)

**Comment by WilliamWDTK** [comment]
One could use a `<progress>` bar, although that would be a different sort of style from what was there before.

**Comment by garethrees** [comment]
Yeah, I don't think it's important to exactly replicate the previous style.

**Comment by garethrees** [comment]
This is desirable, but unlikely to be worked on in the next 12 months so closing for now.
